internal/examples/example-otlp-agent-tempo-grafana/docker-compose.yaml

```diff
@@ -20,7 +30,7 @@ services:
     volumes:
       - ./otel-collector.yaml:/etc/otel-collector.yaml
     ports:
-      - "4317" # otlp grpc
+      - "4317:4317" # otlp grpc
```

Comment:
Update README with instructions on how to run the plugin locally and generate traces?